yt/testing.py

```diff
@@ -600,7 +600,7 @@ def construct_octree_mask(prng=RandomState(0x1D3D3D3), refined=None):
 
 
 def fake_octree_ds(
-    prng=RandomState(0x1D3D3D3),
+    prng=RandomState(0x1D3D3D3),  # noqa B008
```

Comment:
```suggestion
    prng=RandomState(0x4D3D3D3),  # noqa B008
```

Reply:
oh my, I have no clue what this changes does

Reply:
probably best to just hit the accept button on the suggestion then :P